src/dist/replication/meta_server/meta_bulk_load_service.cpp

```diff
@@ -394,7 +394,8 @@ void bulk_load_service::on_partition_bulk_load_reply(error_code err,
         handle_app_downloading(response, primary_addr);
         break;
     case bulk_load_status::BLS_DOWNLOADED:
-        handle_app_downloaded(response);
+        update_partition_status_on_remote_storage(
+            response.app_name, response.pid, bulk_load_status::BLS_INGESTING);
         // when app status is downloaded or ingesting, send request frequently
```

Comment:
why app status is downloaded or ingesting, send request will be more frequently?

Reply:
When app status is downloaded and ingesting, partition status may be ingesting. 
When partition status is updated to ingesting, replica will reject other write requests and recover when primary notice group finish ingesting. Primary will check if group finish ingesting while receive bulk_load_request. As a result, meta send request frequently will let replica recovering write requests more quickly.